### ISSUE TITLE
Custom/Default +1 Button

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Torsten2217/google-plus-platform#564]
```
Describe in detail the new feature you would like to see added
-> What new capabilities would you like to see added to the API?

+1 button customization, or different animations


Describe your use case, or some example use cases, for this feature
->Why do you want this feature, and how might it be valuable not just to
your application, but to many developers?

Currently for a plus one button there is a semi-universal look/feel, similar to 
facebook and twitter like/tweet buttons and how they display counts.

I would like the options to have the animations replicate that on Google+ 
itself, without the bubble around the number, or a way to just send a list of 
URLs and get the +1 counts, so I can display a very customized alternative.


Basically, I want google to manage the +1 counts, and I would like to be able 
to have my site look like Google+ when someone +1s something (either by passing 
an option, or the ability to program for it).
```

Original issue reported on code.google.com by `j...@woodwardmedia.net` on 20 May 2013 at 5:37

Attachments:
- [Untitled-1.png](https://storage.googleapis.com/google-code-attachments/google-plus-platform/issue-564/comment-0/Untitled-1.png)
